force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -1,368 +1,466 @@
+   * @return The sum of all integers up to the limit.
+   */
+  public static Integer sumUpToLimit(Integer intLimit) {
+    Integer sum = 0;
+    for (Integer i = 0; i <= intLimit; i++) {
+      sum += i;
     }
-
-    /**
-     * Question 13
-     * Create a list filled with integers based on the input parameter.
-     * Example: createIntegerList(5) should return [1, 2, 3, 4, 5] 
-     * @param n The last integer in the List.
-     * @return A List of integers from 1 to 'n'.
-     */
-    public static List<Integer> createIntegerList(Integer n) {
-        // Create a new list of integers 
-
-        /*
-        for() {
-            //add integer to the list
-        }
-        */ 
-
-        return null; // Replace null with the variable you used to store the result
+    return sum;
+  }
+
+  /**
+   * Question 9
+   * Generate String "Hello World!" three times, with each occurrence separated by a semicolon.
+   * Example: returnHelloWorld() should return "Hello World!; Hello World!; Hello World!; "
+   * @return The string "Hello World!; Hello World!; Hello World!; "
+   */
+  public static String returnHelloWorld() {
+    // Initialize the result String
+    String result = '';
+    // Use a for loop to append 'Hello World!;' to the result string three times
+    for (Integer i = 0; i < 3; i++) {
+      result += 'Hello World!' + '; ';
     }
-
-    /**
-     * Question 14
-     * Remove duplicate values from the list of Strings.
-     * The returned Set contains the unique strings from the input List.
-     * Example: createStringSet(['a', 'b', 'a', 'c']) should return ['a', 'b', 'c']
-     * @param inputList The List of strings.
-     * @return A Set of unique strings.
-     */
-    public static Set<String> createStringSet(List<String> inputList) {
-        // Create a new Set of strings
-        Set<String> stringSet = new Set<String>();
-
-        //add strings to the set
-
-        return null; // Replace null with the variable you used to store the result
+    return result;
+  }
+
+  /**
+   * Question 10
+   * Generate a string where the input string is repeated the given number of times, with each
+   * repetition separated by a semicolon.
+   * Do not add a semicolon after the last repetition.
+   * Example: repeatString("Hello World!", 3) should return "Hello World!; Hello World!; Hello World!"
+   * @param inputString The string to be repeated.
+   * @param repeatCount The number of times the string should be repeated.
+   * @return The new string with the inputString repeated repeatCount times.
+   */
+  public static String repeatString(String inputString, Integer repeatCount) {
+    // Initialize the result String
+    String result = '';
+
+    // Use a for loop to append the inputString to the result string repeatCount times
+    for (Integer i = 0; i < repeatCount; i++) {
+      if (i == repeatCount - 1) {
+        result += inputString;
+      } else {
+        result += inputString + '; ';
```

Comment:
This is fine here because the logic is simple. If your logic gets more complicated, you might want to consider keeping what is common together and then adding what is different. What I mean by that in this example is to always add inputString to result `result += inputString;` Then check if you are not the last in the loop and only add the "; ". `result += '; ';` That prevents you from duplicating the adding of the inputString. This is example is more simple, but something to consider as logic gets more complicated.